client/cli/go/cmds/cluster.go

```diff
@@ -96,6 +112,60 @@ var clusterCreateCommand = &cobra.Command{
 	},
 }
 
+var clusterSetFlagsCommand = &cobra.Command{
+	Use:   "setflags",
+	Short: "Set flags on a cluster",
+	Long:  "Set flags on a cluster",
+	Example: `  * Disable creation of file volumes on a cluster:
+      $ heketi-cli cluster set --file=false 886a86a868711bef83001
+
+  * Enable the creation of block volumes on a cluster:
+      $ heketi-cli cluster set --block=true 886a86a868711bef83001
+`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		s := cmd.Flags().Args()
+		if len(s) < 1 {
+			return errors.New("Cluster id missing")
+		}
+
+		clusterId := cmd.Flags().Arg(0)
+
+		heketi := client.NewClient(options.Url, options.User, options.Key)
+
+		info, err := heketi.ClusterInfo(clusterId)
+		if err != nil {
+			return err
+		}
+
+		req := &api.ClusterSetFlagsRequest{}
+
```

Comment:
Do we care if they run the command with neither --block=<x> nor --file-<x> ?
Right now, it would be a no-op, but a no-op that "writes" to the server. Maybe printing an error (or a warning) would be better to provide feedback to the user?

Reply:
done. heketi-cli cluster setflags now expects at least one of --file or --block.